package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-red-contrib-ical-events",
-  "version": "2.2.2",
+  "version": "2.2.3",
```

Comment:
There is no need to update this manually 😉 
I'm using the package release-it  for it

Reply:
Reverted.